desktop_version/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM centos:7
+
+# run first to improve caching (other things update more often than SDL2)
+WORKDIR /tmp
+RUN curl -O https://www.libsdl.org/release/SDL2-2.0.14.tar.gz
+RUN tar -xf SDL2-2.0.14.tar.gz
+RUN mkdir SDL2-2.0.14/build
+RUN curl -O https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.0.4.tar.gz
+RUN tar -xf SDL2_mixer-2.0.4.tar.gz
+RUN mkdir SDL2_mixer-2.0.4/build
+
+RUN yum -y install \
+    # SDL2 dependencies, adapted from https://hg.libsdl.org/SDL/file/default/docs/README-linux.md
```

Comment:
A simplification might be what I do for flibitBuild:

```
yum -y install yum-utils
yum -y builddep SDL2
```

Full build setup is here: https://icculus.org/finger/flibitijibibo

Reply:
I was avoiding using EPEL. That might make more sense, though.